src/site/content/en/index.njk

```diff
@@ -128,7 +128,7 @@ eleventyComputed:
     </div>
     <div class="w-grid w-pb--std">
```

Comment:
I think we should turn this file into a layout in the _includes directory, instead of copying it. My concern is keeping it in sync over time. If we make a change to the HTML we'll want that to show up in all languages but it could be easy to forget to copy it over.

For the strings of text that appear in this file I think there are a few options. We could either move all of that text into the YAML front matter or the file and just translate those strings. Or we could create a file of translated strings and use eleventy data to access that file. For example, I think something like this might be possible:

`/_includes/homepage.njk`
```njk
<a class="w-button" href="/blog">
  {% helpers.translate('See all articles', lang) %}
</a>
```

Reply:
We can do this in a separate PR. I don't want to block this change.

Reply:
Filed: https://github.com/GoogleChrome/web.dev/issues/3769